ci/jenkins/test.sh

```diff
@@ -322,7 +322,11 @@ function prepare_env {
 function revert_snapshot_windows {
     WIN_NAME=$1
     echo "==== Reverting Windows VM ${WIN_NAME} ====="
-    govc snapshot.revert -vm ${WIN_NAME} win-initial
+    if [[ $WIN_NAME == *"jumper"* ]]; then
+        govc snapshot.revert -vm ${WIN_NAME} win-initial
+    else
+        govc snapshot.revert -vm ${WIN_NAME} pristine-win-initial
```

Comment:
Does pristine-win-initial include the installed ovs drivers?

Reply:
No it dosen't include the ovs drivers ,. they are installed through the yml

Reply:
Do you mean it's signed driver? Unsigned driver can not be installed in the container.

Reply:
yes I have installed the signed driver using netcfg command

Reply:
It won't block CI since we have enabled the test mode and imported trusted certificate. You can continue updating the documentation on driver installation. @NamanAg30

Reply:
@XinShuYang , @NamanAg30 
Driver installed is same which is available in upstream antrea repository,  so it is not signed then certificate needs to be applied ,
otherwise later versions of Windows display a driver signing dialog box for unsigned drivers that require a system administrator to authorize the installation of the driver, potentially preventing any user without the necessary privileges from installing the driver !